app/templates/application.hbs

```diff
@@ -14,7 +14,7 @@
         </div>
     </div>
 </nav>
+{{pinned-notifications}}
 <div id="content">
```

Comment:
I'm not the guy to wire this up. It's beginning to melt my designer brain. I tried the above suggestion and I didn't have any luck.